### PR TITLE
This adds a geography map to my main webpage and allows the user to select from which continent they are from

### PR DESCRIPTION
**Pull request by SantiAce26**
Hi! So this is supposed to add a geography map that shows where users who have visited the website are from.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -63,4 +58,21 @@ async function getMovieQuote(){
 
 }
 
+function drawRegionsMap() {
+    fetch('/geography-data').then(response => response.json()).then((geoGraphyCount) => {
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/script.js]: Rename `geoGraphyCount` to `geographyCount`.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -63,4 +58,21 @@ async function getMovieQuote(){
...
+        const data = new google.visualization.DataTable();
+        data.addColumn('String','Continent');
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/script.js]: `String` to `string`
Data type in DataTable is lower cased.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -63,4 +58,21 @@ async function getMovieQuote(){
...
+        data.addColumn('Number','People');
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/script.js]: `Number` to `number`

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -35,6 +36,23 @@
...
         <img src = "/images/fire.gif"/>
     </div>
+
+    <div class="bubble">
+        <h1>Where are you from?</h1>
+        <div id="regions-div" style="width: 500px; height: 500px;"></div>
+        <h2>Choose what continent you're from!</h2>
+        <form action="/geography-data" method="POST">
+            <select name="Continent">
+                <option value="North America">North America</option>
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/index.html]: Region mode of Geochart does not support continents, Australia should work. See the Data Format section in [Geochart](https://developers.google.com/chart/interactive/docs/gallery/geochart).

**Review by kuanyiwu — CHANGES_REQUESTED** [review]
This is a cool functionality! The chart should render after you make a few changes.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -59,14 +59,17 @@ async function getMovieQuote(){
...
+        Object.keys(geographyCount).forEach((Country) => {
+            data.addRow([Country, geographyCount[Country]]);
         });
-        const options = {};
+        const options = {colorAxis: {colors: ['#00fcc6', '#05ed71', '#007527']},
+                        backgroundColor: '#81d4fa',
+                        datalessRegionColor: '#8df2bc',
+                        defaultColor: '#f5f5f5',};
 
         const chart = new google.visualization.GeoChart(document.getElementById('regions-div'));
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/script.js:74]: Is the Google Maps Javascript API activated on your project? I've been getting the following error. 
https://developers.google.com/maps/documentation/javascript/error-messages#api-not-activated-map-error

File: portfolio/src/main/webapp/styleAboutMe.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Dekko&family=Lato:wght@900&family=Rock+Salt&display=swap');
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/styleAboutMe.css:1]: The two new css files are duplicated from style.css. You can import style.css into the aboutme or projects pages.

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (SantiAce26) [reply portfolio/src/main/webapp/script.js:74]: I do not have the google maps api activated, let me commit all my files to see if that helps. Very weird though

Comment (kuanyiwu) [reply portfolio/src/main/webapp/script.js:74]: Does the map render when you preview it?

Comment (SantiAce26) [reply portfolio/src/main/webapp/script.js:74]: Yeah it does, I even deployed it and it runs fine on there too

Comment (kuanyiwu) [reply portfolio/src/main/webapp/script.js:74]: Ah okay I see this on deployment.

**Review by kuanyiwu — APPROVED** [review]

**Pull request merged**